orttraining/orttraining/python/training/_utils.py

```diff
@@ -177,3 +178,30 @@ def import_module_from_file(file_path, module_name=None):
     sys.modules[module_name] = module
     spec.loader.exec_module(module)
     return module
+
+def state_dict_model_key():
+    """Returns the model key name in the state dictionary"""
+
+    return 'model'
+
+def state_dict_optimizer_key():
+    """Returns the optimizer key name in the state dictionary"""
+
+    return 'optimizer'
+
+def state_dict_partition_info_key():
+    """Returns the partition info key name in the state dictionary"""
+
+    return 'partition_info'
+
+def state_dict_trainer_options_key():
+    """Returns the trainer options key name in the state dictionary"""
+
+    return 'trainer_options'
+
+def state_dict_precision_map():
+    """Returns the mapping from TensorProto types to the precision type strings in the state_dict"""
+
```

Comment:
since we can have fp16 datatype for frozen weights, we can include that entry here as well

Reply:
I will be removing this check. All frozen weights, irrespective of their types will be considered as full precision model weights and so will be put in the ```fp32``` nucket.